AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -315,6 +315,7 @@ - (void)_initializeInstance
   _environmentState = ASEnvironmentStateMakeDefault();
   
   _calculatedDisplayNodeLayout = std::make_shared<ASDisplayNodeLayout>();
```

Comment:
If we use nullptr for _pending can we do that for _calculated?  Not important now, just pointing it out